src/EnergyPlus/UnitarySystem.cc

```diff
@@ -10891,9 +10923,12 @@ namespace UnitarySystems {
                 Fans::SimulateFanComponents(state, blankString, FirstHVACIteration, this->m_FanIndex, state.dataUnitarySystems->FanSpeedRatio);
             }
         }
-
-        Real64 SuppPLR = this->m_SuppHeatPartLoadFrac;
         if (this->m_SuppCoilExists) {
+            Real64 SuppPLR = 0.0;
+            if (this->m_NumOfSpeedSuppHeating > 0) {
+                this->calcMultiStageSuppCoilStageByLoad(state, SuppCoilLoad, FirstHVACIteration);
+            }
+            SuppPLR = this->m_SuppHeatPartLoadFrac;
```

Comment:
This is the change I was expecting. However, if m_SuppHeatPartLoadFrac was already available here before (i.e., no other supp heating coil types are called here), then is the call to calcMultiStageSuppCoilStageByLoad in the correct place? And if this call were moved to where the other supp heating coils are modeled to find the PLR, then this code could revert to the original? Regardless, if this eliminates the diffs then this type of check on coil type is OK for now.